src/docs/operator-guides/migrating-from-outside-sharetribe/index.md

```diff
@@ -28,33 +32,102 @@ Intermediary is an edn (https://github.com/edn-format/edn) data format
 that allows defining data rows for marketplaces. It supports creating
 links between rows via references.
 
+We recommend that you use an .edn library to generate the data and
+validate the .edn syntax to ensure that the file is up to standard.
+
 ### Format
 
 Intermediary is specified as
 [a map](https://clojure.org/reference/data_structures#Maps) with the
 following keys:
 
-- :ident - identifies the target marketplace by marketplace ident,
+- :ident - identifies the target marketplace by marketplace ID,
 - :data - a seq of marketplace data rows. Each row is a 2-tuple of id
   and content.
 
+You can find your marketplace ID in Flex Console > Build > General. Note
+that the anonymised test file needs to specify your test environment
+marketplace ID and your production data file needs to specify your
+production environment marketplace ID.
+
 The id part of row 2-tuple is specified as a tuple of 1 to 3 elements.
 The first element is always an id attribute and identifies the row type.
 The second element can be an alias or an import id. A 3 element version
 has id attr, import id and an alias.
 
+```
+;; 1 element tuple
+[:im.stripeAccount/id]
+
+;; 2 element tuple with import id
+[:im.image/id #uuid "58afd8e1-e336-4ca4-a1e7-ff1d91856a6c"]
+
+;; 2 element tuple with alias
+[:im.user/id :user/jane]
+
+;; 3 element tuple
+[:im.listing/id #uuid "b074e697-ab0c-4746-a195-c58d73606b1f" :listing/rock-sauna]
+```
+
 Import ids can be given to rows and they should be unique for the
-marketplace and they are of type UUID. Aliases can be used to reference
-rows from other rows. Aliases are useful when the data is written by
-hand. Import ids are useful for programmatically generated / exported
-data. However, note that import ids are only used in the import phase
-and they can not be mapped to existing resource ids. Content part of row
-2-tuple is a map that is processed by tuple expansion function
+marketplace and have type [UUID](#uuid). Aliases can be used to
+reference rows from other rows. Aliases are useful when the data is
+written by hand. Import ids are useful for programmatically generated /
+exported data.
+
+Note that import ids are only used in the import phase and they can not
+be mapped to existing resource ids. TODO: EXPLAIN BETTER! Content part
+of row 2-tuple is a map that is processed by tuple expansion function
 registered for the content type.
 
-If you are not generating data by hand, it is highly recommended that
-you use the 2 element form of the id tuple. This means that you don't
-use aliases but unique uuid's throughout the file to reference entities.
+If you are not generating data by hand (TODO WHAT IS THE JUSTIFICATION
```

Comment:
What's the reasoning behind this recommendation?

Reply:
Good question. At least if data is programatically generated, it makes sense to use UUID generation to avoid ID collision instead of creating a custom alias based ID system.